text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/builtin/GradientTag.java

```diff
@@ -21,28 +21,31 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package net.kyori.adventure.text.minimessage.transformation.inbuild;
+package net.kyori.adventure.text.minimessage.tag.builtin;
```

Comment:
I think we should rename this `standard` rather than `builtin` - I can see a case where we might want to include extra tags that aren't included in the standard tag resolver but are still shipped with MiniMessage (see the colour alias issue for example).